hw_2.py

```diff
@@ -49,4 +152,61 @@
 #     ["X", "X", "O"]
 # ]) ➞ "Draw"
 
+def main(tic_tac_toe):
+    """Function defines who winning in 3x3 matrix of a completed tic-tac-toe game"""
+    game_board = join_array(tic_tac_toe)
+    marks = ['X', 'O']
+    for mark in marks:
+        if win_check(game_board, mark) is True:
+            return f'Win "{mark}"'
+    else:
+        return 'Draw'
+
+
+def join_array(tic_tac_toe):
+    """Function unpacking 3x3 matrix in list"""
+    general_array = [*tic_tac_toe[0], *tic_tac_toe[1], *tic_tac_toe[2]]
+    return general_array
+
+
+def win_check(game_board, mark):
+    """Function check winnig combination in game"""
+    if mark == game_board[0] and mark == game_board[1] and mark == game_board[2]:
+        return True
+    elif mark == game_board[3] and mark == game_board[4] and mark == game_board[5]:
+        return True
+    elif mark == game_board[6] and mark == game_board[7] and mark == game_board[8]:
+        return True
+    elif mark == game_board[0] and mark == game_board[4] and mark == game_board[8]:
+        return True
+    elif mark == game_board[2] and mark == game_board[4] and mark == game_board[6]:
+        return True
+    elif mark == game_board[0] and mark == game_board[3] and mark == game_board[6]:
+        return True
+    elif mark == game_board[2] and mark == game_board[5] and mark == game_board[8]:
+        return True
+    elif mark == game_board[1] and mark == game_board[4] and mark == game_board[7]:
+        return True
+    return False
```

Comment:
I think usage of the logical operator can make your code more compact. 

    if mark == game_board[0] and mark == game_board[1] and mark == game_board[2] or \
            mark == game_board[3] and mark == game_board[4] and mark == game_board[5] or \
            mark == game_board[6] and mark == game_board[7] and mark == game_board[8] or \
            mark == game_board[0] and mark == game_board[4] and mark == game_board[8] or \
            mark == game_board[2] and mark == game_board[4] and mark == game_board[6] or \
            mark == game_board[0] and mark == game_board[3] and mark == game_board[6] or \
            mark == game_board[2] and mark == game_board[5] and mark == game_board[8] or \
            mark == game_board[1] and mark == game_board[4] and mark == game_board[7]:
        return True
    return False